snuba/environment.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+import logging
+import os
+from typing import Optional
+
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.gnu_backtrace import GnuBacktraceIntegration
+
+from snuba import settings
+
+
+def setup_logging(level: Optional[str] = None) -> None:
+    logging.basicConfig(
+        level=getattr(logging, level, settings.LOG_LEVEL).upper(),
+        format="%(asctime)s %(message)s",
+    )
+
+
+def setup_sentry() -> None:
+    sentry_sdk.init(
+        dsn=settings.SENTRY_DSN,
+        integrations=[FlaskIntegration(), GnuBacktraceIntegration()],
```

Comment:
We likely don't need/want the FlaskIntegration for script (that are not running on flask). Does this hurt them ?

Reply:
This is a good point that I had not considered. After a quick skim of the integration code, I don't think that there are any downsides as long as flask is importable: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/flask.py